modules/sdk-coin-ada/src/lib/transactionBuilder.ts

```diff
@@ -282,7 +282,7 @@ export abstract class TransactionBuilder extends BaseTransactionBuilder {
         // support for multi-asset consolidation
         if (this._multiAssets !== undefined) {
           const totalNumberOfAssets = CardanoWasm.BigNum.from_str(this._multiAssets.length.toString());
-          const minAmountNeededForOneAssetOutput = CardanoWasm.BigNum.from_str('1000000');
+          const minAmountNeededForOneAssetOutput = CardanoWasm.BigNum.from_str('1500000');
```

Comment:
What happens if theres more than one asset?

Reply:
We construct a separate output for each asset/token in the UTXO, so each output will have 1.5 based on the new minimum amount